spec.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Semantic Specification
+---
+
+## Concepts and Terminology
+
+A **Trace** represents the potentially distributed, potentially concurrent data/execution path in a (potentially distributed, potentially concurrent) system.
```

Comment:
Would adding "It is a collection of one or more spans." at the end help clarify this a bit?